ui/lib/core/addon/components/search-select-with-modal.hbs

```diff
@@ -20,6 +20,7 @@
       helpText=@helpText
       placeholder=@placeholder
       id=@id
+      selectLimit=@selectLimit
```

Comment:
thanks! missed this when the component was originally created! can you add it to the jsdoc params?